bindings/python/pyiree/compiler/compiler.cc

```diff
@@ -377,6 +375,7 @@ std::shared_ptr<OpaqueBlob> CompilerModuleBundle::Compile(
 
 void CompilerModuleBundle::RunPassPipeline(
     const std::vector<std::string>& pipelines) {
+  registerAllPasses();
```

Comment:
Shouldn't it be sufficient to register the limited set of MLIR dialects used in IREE?
```suggestion
  mlir::registerMlirPasses();
```